### ISSUE TITLE
Add loader or paging

### ISSUE THREAD

**Issue by TheGrandVizier** [Beit-Hatfutsot/wp-charney#20]
Some categories require too long to load.
The clear solution is paging system or a loader - whichever is cheaper and  easier to execute.

Pages such as http://leoncharney.bh.org.il/category/career/ take much too long to load and while they are loading, they look empty and static.

@nirgoldberg what easier, paging or loader?
@nurit FYI

**Comment by nirgoldberg** [comment]
Loader implementation is easier, but I'm not sure it's the best solution, as if the page contains many items, maybe it's better to implement pagination and earn both enhanced loading time and better presentation.

**Comment by nirgoldberg** [comment]
Another good solution is "Load more" button for each bulk of X items.

**Comment by TheGrandVizier** [comment]
No. That  is the worst option available. Because if the user wants to get back to something  he saw down the list, he has to click "load  more" a bunch of times to find it again. No more Load more please.
Lets do paging, then.
